src/task/task.json

```diff
@@ -66,12 +66,15 @@
     "metrics.codeMetrics.titleSizeXS": "XS",
     "metrics.codeMetrics.titleTestsInsufficient": "⚠️",
     "metrics.codeMetrics.titleTestsSufficient": "✔",
-    "metrics.codeMetricsCalculator.insufficientAzureReposAccessTokenPermissions": "Could not access the resources. Ensure 'System.AccessToken' has access to 'Code' > 'Read' and 'Pull Request Threads' > 'Read & write'.",
-    "metrics.codeMetricsCalculator.insufficientGitHubAccessTokenPermissions": "Could not access the resources. Ensure 'System.AccessToken' has access to 'repos'.",
+    "metrics.codeMetricsCalculator.insufficientAzureReposAccessTokenPermissions": "Could not access the resources. Ensure the PR Metrics 'System.AccessToken' has access to 'Code' > 'Read' and 'Pull Request Threads' > 'Read & write'.",
```

Comment:
It is not always System.AccessToken being used. As per my other comment, that is a special "god-mode" OAuth token in ADO whereas the auth token being used may actually be a scoped PAT.
If it's not feasible to change the SYSTEM_ACCESSTOKEN references throughout, the user-facing messaging should at least be changed to reference "the Access Token provided" or similar instead of 'System.AccessToken' since specifically mentioning System.AccessToken in the context of an ADO pipeline is misleading the user into thinking that the PCBS account is being used to execute the task.